cit3.0-web/src/components/UserListItem/UserListItem.js

```diff
@@ -0,0 +1,51 @@
+import { Row, Col, Button } from "react-bootstrap";
+import PropTypes from "prop-types";
+import { formatDate } from "../../helpers/helpers";
+
+const UserListItem = ({ user, handelEditAction, handleDeleteAction }) => (
+  <div key={user.id} className="user-table-row w-100 p-2">
+    <Row>
+      <Col>{user.name}</Col>
+      <Col>{user.email}</Col>
+      <Col>
+        {user.municipalities.length
+          ? user.municipalities.map((muni, index) => (
+              <p key={index}>{muni.name}</p>
+            ))
+          : null}
+        {user.regionalDistricts.length
+          ? user.regionalDistricts.map((RD, index) => (
+              <p key={index}>{RD.name}</p>
+            ))
```

Comment:
Same question as above for keys, can we use something like `RD.id`?

Reply:
There can be a collision, there should just be a prefix to index or id